dist/build.js

```diff
@@ -1692,7 +1702,10 @@ Branch.prototype.banner = wrap(callback_params.NO_CALLBACK, function(a, b, c) {
   this.closeBannerPointer = banner(this, d, c, this._storage);
   a();
 });
-Branch.prototype.closeBanner = wrap(0, function(a) {
+Branch.prototype.setBranchViewData = wrap(callback_params.NO_CALLBACK, function(a, b) {
+  this._bvData = goog.object.clone(b || {});
+});
+Branch.prototype.closeBanner = wrap(callback_params.NO_CALLBACK, function(a) {
```

Comment:
The change from `wrap(0` to `wrap(callback_params.NO_CALLBACK` is not entirely necessary here, but makes for more consistent code